client/package.json

```diff
@@ -7,8 +7,6 @@
     "dependencies": {
         "superdesk-analytics": "github:tomaskikutis/superdesk-analytics#next",
```

Comment:
This can now change to use the official analytics repo

Reply:
which branch?

Reply:
The other repos are using the release branch, so I would suggest the same
`github:superdesk/superdesk-analytics#release/1.33`